src/frontend/components/UI/DownloadToastManager/index.tsx

```diff
@@ -220,6 +221,28 @@ export default function DownloadToastManager() {
     return 'inProgress'
   }
 
+  function handlePauseDownload() {
+    trackDownloadStatusChange({
+      event: 'Game Install Paused',
+      properties: {
+        store_name: runner,
+        game_title: title
+      }
+    })
+    window.api.pauseCurrentDownload()
+  }
+
+  function handleResumeDownload() {
+    trackDownloadStatusChange({
+      event: 'Game Install Resumed',
+      properties: {
+        store_name: runner,
+        game_title: title
+      }
+    })
+    window.api.resumeCurrentDownload()
+  }
+
```

Comment:
Would this be easier to handle this in the main process?

Reply:
I decided to do it on the frontend because then we can track the actually interaction instead of what the backend processed. I believe that it makes more sense to have data about what the user wanted to do here instead of what how the app behave (since this we use Sentry already). But I am not data expert.

Reply:
I just really think this tracking should be in the main process (hyperplay store manager) with the rest of the logic and the frontend should just be for frontend state and react components (basically View Manager and View in MVVM model). Within that framework,  metrics logic is business logic that should be in the main process not in the view or view manager.

We will forget to add this when we add the pause button on the game page for instance. If it's in the main process, it is defined in one place and doesn't need updating. We can also mock this out and write tests more easily (and in one place) in the main process. It's more testable and maintainable.

If the app is not behaving in accordance with the user interaction, then our e2e or manual tests aren't working. We shouldn't incur tech debt to handle the case where that happens. We should just fix the functionality

Also do you think we'll expose pause, cancel, resume through the store or hp protocol sometime in the future? These metrics would not fire in that case if we leave them in the frontend

I'll let @red-game-dev and @eliobricenov weigh in here too

Reply:
Ok, I will move it to the backend, no problem.
Although I don't agree that this is business logic and tracking what the user is doing makes more sense than the call per-se since we already have Sentry to catch issues on that.
But we need this merged so if we keep discussing here I don't think this will happen :P

Reply:
Done